downstream/titles/release-notes/topics/installer-24-6.adoc

```diff
@@ -0,0 +1,40 @@
+// This is the release notes file for AAP 2.4 async installer release 2.4-6 dated March 01, 2024
+
+[id="installer-24-6"]
+
+= RHSA-2024:1057 - installer release 2.4-6 - March 01, 2024
+
+link:https://access.redhat.com/errata/RHSA-2024:1057[RHSA-2024:1057]
+
+
+//Automation hub
+== {HubNameStart}
+
+.New features and enhancements
+
+* Display the download count for each collection in {HubName} (AAP-18298)
+
+//Event-Driven Ansible
+== {EDAName}
+
+.New features and enhancements
+
+* Added a parameter to control the number of running activations per {EDAName} worker service (AAP-20672)
+
+.Bug fixes
+
+* Added `EDA_CSRF_TRUSTED_ORIGINS` which can be set by user input or defined based on the allowed hostnames that are determined by the installer (AAP-20244)
+
+* {EDAName} installation now fails when the pre-existing {ControllerName} version is 4.4.0 or older (AAP-20241)
+
+* Added the `podman_containers_conf_logs_max_size` variable for containers.conf to control the max log size for Podman installations. The default value is 10 MiB (AAP-19775) 
+
+* Setting the {EDAName} debug flag to false will now correctly disable Django debug mode (AAP-19577)
```

Comment:
Avoid use of future tense "will"?: 

"Setting the {EDAName} debug flag to false now correctly disables..."